16/main.js

```diff
@@ -0,0 +1,134 @@
+const API = "7106edeb";
+const movieTitleInputField = document.getElementById("name");
+const mainDiv = document.getElementById("main");
+const childDiv = document.createElement("div");
+mainDiv.append(childDiv);
+const buttonSearchFilm = document.getElementById("addbutton");
+const navbarList = document.getElementsByClassName("pagination-container");
+const navbar = navbarList[0];
+let counterForMoreInfoButton = 0;
+let counterForSearchButton = 0;
+let currentPage = 1;
+let startPoint = currentPage;
+function getMovieInfo(movieTitle, page) {
+  fetch(`http://www.omdbapi.com/?s=${movieTitle}&apikey=${API}&page=${page}`, {
+    method: "GET",
+  })
+    .then((res) => res.json())
+    .then((json) => {
+      if (json.Error) {
+        movieTitleInputField.value = json.Error;
+        return 0;
+      }
+      let totalNUmberOfPages = +Math.ceil(+json.totalResults / 10);
```

Comment:
Looks like it is broken

https://user-images.githubusercontent.com/84010501/203498595-491fd135-c210-4c5e-b79f-007661e0aa2e.mov